app/models/issue.rb

```diff
@@ -52,68 +62,25 @@ def to_s
 
   alias logging_subject_name to_s
 
-  def map_icon
-    "icons/map/active/png/#{kind || 'blank'}-#{icon_color}.png"
-  end
-
-  def list_icon
-    "icons/list/png/#{kind || 'blank'}-#{icon_color}-22px.png"
-  end
-
-  private
-
-  def author_blacklist
-    return if errors[:author].present?
-    return unless MailBlacklist.where(pattern: [author, author[author.index('@') + 1..]]).any?
-    errors.add :author, :blacklist_pattern
-  end
-
-  def icon_color
-    case status
-    when 'received', 'reviewed'
-      'red'
-    when 'in_process'
-      'yellow'
-    else
-      'gray'
-    end
-  end
-
-  def add_photo
-    return if new_photo.blank?
-    photos.new file: new_photo, author: Current.user.to_s, status: :internal
-  end
-
-  def update_address_parcel_property_owner
-    self.address = Geocodr.address(self)
-    self.parcel = Geocodr.parcel(self)
-    self.property_owner = Geocodr.property_owner(self)
-  end
-
-  def set_confirmation_hash
-    self.confirmation_hash = SecureRandom.uuid
+  def archived
```

Comment:
```suggestion
  alias archived archived_at?
```
`archived_at?` wird mit 121f2aa eingeführt.